java/maven/cloudbuild.yaml

```diff
@@ -0,0 +1,8 @@
+# In this directory, run the following command to build this builder.
+# $ gcloud container builds submit . --config=cloudbuild.yaml
+
+steps:
+- name: 'gcr.io/cloud-builders/docker'
+  args: ['build', '--tag=gcr.io/$PROJECT_ID/maven', '.']
```

Comment:
(this is a new theme which I want to begin - makes it easier to sort/organize all the tool builders for each language)